packages/storybook/src/example-pages/toepassingen/aanvragen/verzoeken.ts

```diff
@@ -0,0 +1,123 @@
+import { html } from "lit-html";
+
+import { examplePageFactory } from "../../../example-page-factory";
+import { alerts } from "./verzoeken.content";
+
+examplePageFactory(
+  "Toepassingen/Aanvragen",
+  "Verzoeken",
+  ({
+    applicationHeadingTemplate,
+    alertTemplate,
+    formButtonsTemplate,
+    formTemplate,
+    formGroupInputTemplate,
+    formGroupSelectTemplate,
+    formGroupStaticTemplate,
+  }) => html`
+    <div class="container">
+      <main>
+        <form class="form-horizontal">
+          ${applicationHeadingTemplate({
+            title: "Aanvraag Laan van Eik en Duinen 125, Den Haag",
+            subtitle: "7. Verzoeken indienen",
+            step: "Stap 7/7",
+          })}
+          ${alerts.map((alert) => html` ${alertTemplate(alert)} `)}
+          ${formTemplate({
+            mode: "horizontal",
+            formGroups: html`
+              ${formButtonsTemplate({
+                buttons: [{ type: "button", variant: "tertiary", label: "Ergens anders indienen" }],
+              })}
+              <fieldset>
```

Comment:
Hm dit moeten we ook omschrijven naar een data driven modelering

Reply:
Ik weet alleen even niet hoe, er zitten namelijk fieldsets in en die worden niet ondersteunt in het FormGroupModel

Reply:
Ohja.. Naja, andere keer weer